Auth0/AuthenticationError.swift

```diff
@@ -27,15 +27,9 @@ public struct AuthenticationError: Auth0APIError, @unchecked Sendable {
     /// HTTP status code of the response.
     public let statusCode: Int
 
-    /// The underlying `Error` value, if any. Defaults to `nil`.
-    public var cause: Error? {
```

Comment:
This was moved into `Auth0APIError`, so it's available for all API error types, not just `AuthenticationError`.